<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QDialog" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>553</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>634</width>
    <height>553</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>634</width>
    <height>553</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QComboBox" name="accountComboBox">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>250</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="accessibleName">
    <string>login_username</string>
   </property>
   <property name="locale">
    <locale language="Chinese" country="China"/>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::NoInsert</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>300</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="rememberPsd">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>370</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>记住账号</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="autoLogin">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>370</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>410</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>35</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>登  录</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLog">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>260</width>
     <height>44</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelVersion">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>50</width>
     <height>44</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButtonForgetPassword">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>63</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>忘记密码?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
